app/src/app/Marketplace.tsx

```diff
@@ -1,19 +1,29 @@
-import React from 'react'
-import { Route as RouterRoute } from 'react-router-dom'
+import React, { FunctionComponent } from 'react'
+import { Redirect, Route as RouterRoute, useParams } from 'react-router-dom'
 import withErrorBoundary from '$shared/utils/withErrorBoundary'
 import ErrorPage from '$shared/components/ErrorPage'
-import ProductPage from '$mp/containers/ProductPage'
+import ProjectPage from '$mp/containers/ProjectPage'
 import StreamPreviewPage from '$mp/containers/StreamPreviewPage'
 import ProjectsPage from '$mp/containers/Projects'
+import ProjectConnectPage from '$mp/containers/ProjectPage/ProjectConnectPage'
+import ProjectLiveDataPage from '$mp/containers/ProjectPage/ProjectLiveDataPage'
 import NewProductPage from '$mp/components/NewProductPage'
 import routes from '$routes'
 const Route = withErrorBoundary(ErrorPage)(RouterRoute)
 
+const ProjectDetailsPageRedirect: FunctionComponent = () => {
```

Comment:
BTW, I have used `React.FC` which is a shorthand for `FunctionComponent`. Saves a few keyboard clicks and an import 😄  Just FYI.

Reply:
TBH I prefer FunctionComponent because it literally says what it is :D